src/main/resources/templates/index.html

```diff
@@ -13,7 +13,7 @@
 
     <ul class="nav">
         <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
+        <li><a href="AddFacility.html">Add Facility</a></li>
```

Comment:
This spelling error came up in a previous review.  When that occurs, first in wins.